includes/Dashboard/Settings.php

```diff
@@ -125,28 +131,12 @@ public function render_settings_page() {
 	 * @return void
 	 */
 	public function check_notice_stripe() {
-		$message = false;
 		// phpcs:disable
 		if ( empty( $_GET['notices-code'] ) || isset( $_GET['settings-updated'] ) ) {
 			return;
 		}
-		switch ( $_GET['notices-code'] ) {
-			case 'payment':
-				$message = __( 'Payment successful', 'wubtitle' );
-				break;
-			case 'update':
-				$message = __( 'Payment information updated', 'wubtitle' );
-				break;
-			case 'reset':
-				$message = __( 'License key sent, check your email!', 'wubtitle' );
-				break;
-			case 'delete':
-				$message = __( 'Unsubscription successful', 'wubtitle' );
-				break;
-			case 'reactivate':
-				$message = __( 'Reactivation of the plan successful', 'wubtitle' );
-				break;
-		}
+		$helpers= new Helpers();
+		$message = $helpers->switch_message($_GET['notices-code']);
```

Comment:
La classe helper non serve, riscrivilo in questo modo:
```suggestion
<?php

$notice_code = [
	'payment' => __('Payment succesful'),
	'update' => __('Payment succesful'),
	'reset' => __('Payment succesful'),
	//...
];
$notice_code = sanitize_text_field( $_GET[] );
$message = $notice_code[$notice_code];
```